app/assets/javascripts/dashboard.js

```diff
@@ -37,54 +34,50 @@ function hide_widget(item){
 
 }
 
-function save_position(){
-    var positions = JSON.stringify(serialize_grid());
-    localStorage.setItem("grid-data", positions);
-    window.location.reload();
+function remove_widget(item){
+    var widget = $(item).parents('li.gs_w');
+    var gridster = $(".gridster>ul").gridster().data('gridster');
+    if (confirm(__("Are you sure you want to delete this widget from your dashboard?"))){
+        $.ajax({
+            type: 'delete',
+            url: $(item).data('url')
+        });
+        gridster.remove_widget(widget);
+    }
+}
+
+function save_position(path){
+
+    var positions = serialize_grid();
+    $.ajax({type: 'POST',
+            url: path,
+            data: {'widgets': positions},
+            success: function(){
+                $.jnotify(__("Widget positions successfully saved to database"), 'success', false);
```

Comment:
Remove the `to database` part here and in the error message. Users don't need to know the details. Also isn't it `position` ? It's **one** position defined by **multiple** coordinates I'd say.